paddle/fluid/pybind/pybind.cc

```diff
@@ -174,6 +174,28 @@ PYBIND11_MODULE(core_noavx, m) {
 
   m.def("set_num_threads", &platform::SetNumThreads);
 
+  m.def("from_dlpack", [](py::capsule *dltensor) {
+    DLManagedTensor *dmt = reinterpret_cast<DLManagedTensor *>(
+        PyCapsule_GetPointer(dltensor->ptr(), "dltensor"));
+    PyCapsule_SetName(dltensor->ptr(), "used_dltensor");
```

Comment:
Why call `PyCapsule_SetName` here to change the name? BTW, what is the meaning of `dltensor` and `used_dltensor`?

Reply:
Refer to cupy's and cudf's unified implemention, PyCapsule_GetPointer method get capsule object by the name attribute, so we need to be consistent with cudf and cupy by using name 'dltensor'.

And refer to cupy and cudf, after we used a dltensor, need to Make sure this capsule will never be used again, so rename the tensor capsule object name to 'used_dltensor'.
I don't exactly the real reason, but we need to be cudf.

This is consistent with cupy and cudf.

https://github.com/cupy/cupy/blob/833a38b5f797c9d44548847f17469117dc8856a4/cupy/core/dlpack.pyx#L149

https://github.com/rapidsai/cudf/blob/388dd12a01a3099d5eb63191d2628a2f2918c0bd/python/cudf/cudf/_lib/dlpack.pyx#L136